### ISSUE TITLE
3D Photo projection inside world orbs

### ISSUE THREAD

**Issue by AmbientSkys** [Neos-Metaverse/NeosPublic#3717]
### Is your feature request related to a problem? Please describe.

One thing that I've always thought would be cool in any vr game with different worlds/realms was the ability to peak into the world before joining, such as a 3D Photo or other.

### Describe the solution you'd like

So what I though would be a cool Idea is the ability to look into world orbs, and instead of them just being a 3D projection on the outside, they expand around the viewpoint of a user like a 3D Photo, sort of like joining the world in "Demo" mode before actually joining. This would allow users to get a feel for a world and maybe a few users already in said world before joining.  

### Describe alternatives you've considered

a 360 photo would work just as well, though would be less preferable.

**Comment by H3BO3** [comment]
Related to #505

**Comment by AmbientSkys** [comment]
> Related to #505

Isn't #505 related to the camera taking 360 photos? 
Mine is about world orbs, containing a 360 photo of the current world state.

**Comment by LexiBasilisk** [comment]
#505 would be a requisite for this issue- because Neos would first need a means to capture a stereo 360 screenshot to use as the thumbnail for the world orb. Assuming that is what you mean by "3D" photo at least.

**Comment by AmbientSkys** [comment]
Ah, I misworded I suppose. But maybe I can add as an alternative. I meant 360 photo since it doesn't necessarily have to be 3D. But, that could be an alt.

**Comment by shadowpanther** [comment]
World orbs are already 360 projections of the "preview" of the world. They are just usually too small to really get a look at.

**Comment by AmbientSkys** [comment]
> World orbs are already 360 projections of the "preview" of the world. They are just usually too small to really get a look at.

This is why I wish it would wrap around the user locally instead. Would be more immersive than squinting at an orb

**Comment by shadowpanther** [comment]
You technically can still do it if you scale the world orb up. The problem is, if you touch it twice with your finger, you'll launch it.
Or someone could create a _something_ that accepts the world orb and creates a surround preview for you. Should be possible, I think.

**Comment by AmbientSkys** [comment]
> You technically can still do it if you scale the world orb up. The problem is, if you touch it twice with your finger, you'll launch it. Or someone could create a _something_ that accepts the world orb and creates a surround preview for you. Should be possible, I think.

Thats true, though having it be a native feature would be more preferable. So i don't have to spawn a tool all the time, or permanently attach something to my model
